africanlii/views/documents.py

```diff
@@ -56,5 +56,8 @@ def render_to_response(self, context, **response_kwargs):
                 file = open(f"{temp_dir.name}/{filename}", "rb")
             else:
                 file = self.object.source_file.file.open()
-            return view_attachment(file)
+            return HttpResponse(
+                file.read(), content_type=self.object.source_file.mimetype
```

Comment:
If we've converted the docx to a pdf, then the content type must be for the pdf, not the docx.

Reply:
Or, put another way, it should always be a PDF.

I still think line 54 should check for a non-PDF and convert it to PDF, rather than checking for docx specifically. Because `.doc` also needs to be converted.